src/main.ts

```diff
@@ -20,6 +21,20 @@ library.add(fas, far)
 loadFonts()
 
 const app = createApp(App)
+
+// Only track usage statistics if the user has not opted out
+if (window.localStorage.getItem('cockpit-enable-usage-statistics-telemetry')) {
+  Sentry.init({
+    app,
+    dsn: 'https://5176a29493da4d31a6384c55d291cd3d@o4507692193415168.ingest.us.sentry.io/4507692196036608',
+    integrations: [Sentry.browserTracingIntegration(), Sentry.replayIntegration()],
+    tracesSampleRate: 1.0, //  Capture 100% of the transactions
```

Comment:
the tracesSampleRate is likely to be the first thing to deplete, and getting 100% does not give much benefit anyway. I´m using it at 0.1 and is already insightful: 

![image](https://github.com/user-attachments/assets/3db129a4-a3b9-4db0-b1b5-bb1062f4167c)

Reply:
there is also the option for sampling the errors ( in python it's usually the exceptions ), might be wise to start with a sampled amount if the app produces exceptions very often

https://docs.sentry.io/platforms/javascript/guides/vue/configuration/options/#sample-rate

Reply:
Nice catch!
Reduced the trace to 10%.

We are trying not to have unhandled errors, so I think for those it's beneficial to keep 100%.